code/modules/admin/verbs/request_music.dm

```diff
@@ -0,0 +1,29 @@
+/mob/verb/request_music()
+	set category = "OOC"
+	set name = "Request Music"
+
+	if(GLOB.say_disabled) //This is here to try to identify lag problems
+		to_chat(usr, span_danger("Speech is currently admin-disabled."), confidential = TRUE)
+		return
+
+	var/msg = tgui_input_text(usr.client, "Please Input a URL", "Music Request", "")
+	if(!msg)
+		return
+	log_prayer("[src.key]/([src.name]): [msg]")
```

Comment:
This should be in a separate log type, it's not a prayer.

Reply:
Also no `src`

Reply:
So for clarification, as this would also add on and make it a config update too, you sure that's desirable? Its still in the same vein as being a prayer, I dont see why the logging could be the same when players already sometimes pray for music regardless?

Reply:
You don't need a config for every logging type